### ISSUE TITLE
Duplicate messages until page reload

### ISSUE THREAD

**Issue by benibr** [thelounge/thelounge#4448]
- _Node version:_ v12.22.5
- _Browser version:_ 96.0.2 (64-bit)
- _Device, operating system:_ Arch Linux
- _The Lounge version:_ 4.3.0

---

I use The Lounge in private mode with LDAP accounts. When I newly login every message I sent, is posted double in the channel.
Also when I connect to a new network, The Lounge create two new connections to the same network.
Also happens with chrome.

Fix is always to reload the tab. No new login required, The Lounge behave normally afterwards.

Logs don't say anything.

I found "Fixed a rare bug in irc-framework that caused duplicate messages" in the changelog of 4.2.0 but could figure out how to further investigate.

**Comment by brunnre8** [comment]
Is that reproducible?

**Comment by benibr** [comment]
Yes it is. Just spawned a new VM and installed TheLounge again. Still exactly the same.

When I add a new network connections I get two entries in the networks list, both connect and if post into a channel the message is sent double.
Reload page -> all fine

**Comment by benibr** [comment]
Edit: TL is running on a VM with Debian 11
Client is Arch Linux laptop

**Comment by benibr** [comment]
After debugging a bit with support of bookworm from IRC, we figured out, that the problem seems to occur only when LDAP Login is enabled.
When local accounts are used, everything works fine.
